diff --git a/node_modules/iam-client-lib/dist/src/cacheServerClient/cacheServerClient.types.d.ts b/node_modules/iam-client-lib/dist/src/cacheServerClient/cacheServerClient.types.d.ts
index bec1305..63b10f8 100644
--- a/node_modules/iam-client-lib/dist/src/cacheServerClient/cacheServerClient.types.d.ts
+++ b/node_modules/iam-client-lib/dist/src/cacheServerClient/cacheServerClient.types.d.ts
@@ -2,7 +2,7 @@ import { IDIDDocument } from "@ew-did-registry/did-resolver-interface";
 import { IRoleDefinition, IAppDefinition, IOrganizationDefinition } from "@energyweb/iam-contracts";
 import { IClaimIssuance, IClaimRejection, IClaimRequest } from "../iam";
 export interface IRole {
-    uid: string;
+    id: string;
     name: string;
     namespace: string;
     owner: string;
@@ -10,7 +10,7 @@ export interface IRole {
     isOwnedByCurrentUser?: boolean;
 }
 export interface IOrganization {
-    uid: string;
+    id: string;
     name: string;
     namespace: string;
     owner: string;
@@ -21,7 +21,7 @@ export interface IOrganization {
     isOwnedByCurrentUser?: boolean;
 }
 export interface IApp {
-    uid: string;
+    id: string;
     name: string;
     namespace: string;
     owner: string;
